test/test-fs-poll.c

```diff
@@ -56,7 +56,8 @@ static void touch_file(const char* path) {
   FILE* fp;
   int i;
 
-  ASSERT((fp = fopen(FIXTURE, "w+")));
+  fp = fopen(path, "w+");
+  ASSERT_NOT_NULL(fp);
```

Comment:
Spot the buglet of using `FIXTURE` instead of `path` here. Harmless because everything calls it with `FIXTURE` but still.